broker/lib/fabrik/Agent.js

```diff
@@ -187,18 +187,20 @@ class Agent extends HttpClient {
       .then(ip => this.post(ip, 'lifecycle/preupdate', context, 200, agentCredsBeforeUpdate));
   }
 
-  createCredentials(ips, parameters) {
+  createCredentials(ips, parameters, preBindResponse) {
     const body = {
-      parameters: parameters
+      parameters: parameters,
+      actions: preBindResponse
```

Comment:
Can you please make necessary changes in agent api doc [here](https://cloudfoundry-incubator.github.io/service-fabrik-broker/api/agent_v1.1.html#credentials-create-credentials-post) and [here](https://cloudfoundry-incubator.github.io/service-fabrik-broker/api/agent_v1.1.html#credentials-delete-credentials-post)

Reply:
How do I edit those files?

Reply:
@steinri [This](https://github.com/cloudfoundry-incubator/service-fabrik-broker/tree/gh-pages) is the doc branch.
You will have to make changes in [following](https://github.com/cloudfoundry-incubator/service-fabrik-broker/tree/gh-pages/src/content/api/agent/v1.0) files.
Steps for building doc pages locally are present in README